src/configure/configure.ts

```diff
@@ -251,6 +251,10 @@ class PipelineConfigurer {
             }
         }
 
+        // Set working directory relative to repository root
+        let gitRootDir = await this.localGitRepoHelper.getGitRootDirectory();
+        this.inputs.pipelineParameters.workingDirectory = path.relative(gitRootDir, this.workspacePath);
```

Comment:
What would be the workingFirectory if root and workspace dir are same? Should we leave it empty in that case?

Reply:
In case when they are same, workingDirectory will be empty, empty string will be returned ''. As this is being handled, I didn't add if condition to check if they are same